spec/models/tslr_claim_spec.rb

```diff
@@ -146,11 +155,27 @@
   end
 
   context "when saving in the “bank-details” validation context" do
-    context "that has bank account details" do
-      it "validates that the values are present" do
-        expect(TslrClaim.new).not_to be_valid(:"bank-details")
-        expect(TslrClaim.new(account_sort_code: "123456", account_number: "87654321")).to be_valid(:"bank-details")
-      end
+    it "validates that the bank_account_number and bank_sort_code are present" do
+      expect(TslrClaim.new).not_to be_valid(:"bank-details")
+      expect(TslrClaim.new(bank_sort_code: "12 34 56", bank_account_number: "87 65 43 21")).to be_valid(:"bank-details")
+    end
+  end
+
+  describe "#bank_account_number" do
+    it "strips out whitespace and the “-” character" do
```

Comment:
💯 I like this approach over removing all non-digit characters

Reply:
👍